tests/unit/Certify.spec.ts

```diff
@@ -70,8 +70,6 @@ function createComponent (
   certifiedBy: string | undefined = undefined,
   date: string = defaultDate,
   certifyStatementResource: CertifyStatementIF = {
```

Comment:
Type doesn't exist any more.

The fields below weren't used right?

Reply:
Sorry, no it doesn't. Will be working in the unit tests more wholly when the configuration is complete.